src/components/pages/GamePlay/GameReadStep.js

```diff
@@ -29,8 +42,30 @@ export default function GameReadStep(props) {
     history.push(`${props.baseURL}/draw`);
   };
 
+  // get API base URL
+  const myStory = () => {
+    const APIURL = getApiUrl();
+
+    // initialize axios call
+    axios
```

Comment:
Should the axios call be made in the api index file? Just to keep them all in the same place. A function could be made in the api index file that also calls getApiUrl.